### ISSUE TITLE
Spoiled state problem

### ISSUE THREAD

**Issue by andrewnazarov** [mumoshu/terraform-provider-helmfile#16]
In https://github.com/mumoshu/terraform-provider-helmfile/issues/9 we decided that taking file approach like
```
resource "helmfile_release_set" "mystack" {
    content = file("./helmfile.yaml")
...
}
```
would fix all the troubles with the stored stated. But we've just faced the other one.

Somehow `terraform apply` failed (the chart wasn't uploaded to the chart museum) and it left a broken state. Also we then realised that the chart had an error in values which were not detected by the linter. Further fixes of a chart and values didn't help since Terraform tries to run diff against its state first. And now we are just locked down - nothing can be done without manipulating with the state.

The question is if it's ok that the state was updated after `terrafrom apply` failed

**Comment by mumoshu** [comment]
@andrewnazarov I thought it might be the nature of terraform providers.

Perhaps we can just ignore `helmfile diff` error and only rely on the output of successful diff outputs to detect changes?

**Comment by mumoshu** [comment]
I'm wondering if #17 can fix this.

**Comment by mumoshu** [comment]
@andrewnazarov I'm closing as resolved via #17. I'd appreciate it if you could test it out!
